compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -2967,13 +2967,13 @@ declare_lint_pass! {
         MISSING_ABI,
         INVALID_DOC_ATTRIBUTES,
         SEMICOLON_IN_EXPRESSIONS_FROM_MACROS,
-        DISJOINT_CAPTURE_MIGRATION,
+        RUST_2021_INCOMPATIBLE_CLOSURE_CAPTURES,
         LEGACY_DERIVE_HELPERS,
         PROC_MACRO_BACK_COMPAT,
-        OR_PATTERNS_BACK_COMPAT,
+        RUST_2021_INCOMPATIBLE_OR_PATTERNS,
         LARGE_ASSIGNMENTS,
-        FUTURE_PRELUDE_COLLISION,
-        RESERVED_PREFIX,
+        RUST_2021_PRELUDE_COLLISIONS,
+        RUST_2021_TOKEN_PREFIXES,
```

Comment:
I guess we have agreed to rename this, right?

Reply:
Yes I will rename to `rust_2021_prefixes_incompatible_syntax`